README.md

```diff
@@ -121,11 +97,12 @@ See CITATION.cff file as well as the following references.
 
 ### Prerequisite
 
-PyNumDiff requires common packages like `numpy`, `scipy`, `matplotlib`, `pytest` (for unittests), `pylint` 
```

Comment:
`pytest` and `pylint` actually aren't required to run the code, so I've put them in the non-essential section.